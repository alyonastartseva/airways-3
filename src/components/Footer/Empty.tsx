import { Box, Link, Text, Image, Flex } from "@chakra-ui/react";
import React from "react";
import googleplay from "@images/googleplay.png";
import appstore from "@images/appstore.png";

export const Empty = () => {
  return (
    <>
      <Box display={"flex"} columnGap="1.25rem" fontSize={".875rem"}>
        <Link href="#" target={"_blank"}>
          About us
        </Link>
        I
        <Link href="#" target={"_blank"}>
          Terms and Conditions
        </Link>
        I
        <Link href="#" target={"_blank"}>
          Contact Us
        </Link>
      </Box>
      <Flex alignItems="center" columnGap="3.875rem">
        <Text fontWeight={"700"}>UX AIR APP</Text>
        <Flex columnGap="1.25rem">
          <Link href="#" target={"_blank"}>
            <Image src={appstore} />
          </Link>
          <Link href="#" target={"_blank"}>
            <Image src={googleplay} />
          </Link>
        </Flex>
      </Flex>
    </>
  );
};
