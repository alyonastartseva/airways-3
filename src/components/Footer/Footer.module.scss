@import '../../styles/variables.scss';
.container {
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 80px;
  padding-right: 30px;
  padding-left: 30px;

  background-color: var(--container-bg-color);

  &.dark-theme {
    background-color: var(--dark-theme-bg-color);
    color: var(--container-color);
  }

  &.dark-theme {
    background-color: #3b3a3a;
    color: #ffffff;
  }

  &.dark-theme {
    background-color: #3b3a3a;
    color: #ffffff;
  }

  & .links {
    display: flex;
    gap: 35px;

    & .link {
      color: var(--container-color);

      font-size: 0.9rem;

      &:hover {
        opacity: 0.8;
        text-decoration: underline;
      }
    }
  }
}
