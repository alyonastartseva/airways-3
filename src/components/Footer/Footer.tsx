import { Link as routerLink } from 'react-router-dom';
import { Box, Flex, Link, Spacer } from '@chakra-ui/react';

import { ELinks } from '@services/constants';
import { useAuth } from '@/hooks';
import { WebsiteLogo } from '@/common';

const Footer = () => {
  const { isAdmin: isLogged } = useAuth();
  // TODO заглушка для отображения контента для неавторизованного пользователя
  const { pathname } = window.location;
  const isSignIn = pathname === ELinks.AUTHORIZATION;

  const hoverStyles = {
    color: 'white',
    opacity: '0.8',
    textDecoration: 'underline',
  };
  return (
    <Box
      h="80px"
      display="flex"
      bg="#445EBD"
      pr="30px"
      pl="30px"
      borderTopColor={'#C2C2C2'}
      borderTopWidth={1}
      justifyContent="space-between"
      alignItems="center"
    >
      {isLogged && !isSignIn ? (
        <Flex gap="1rem" color="#FFFFFF" fontSize="0.9rem" alignItems="center">
          <Link as={routerLink} _hover={hoverStyles} to="/">
            Air Alien © 2023
          </Link>
        </Flex>
      ) : (
        <Flex gap="1rem" color="white" fontSize="0.9rem" alignItems="center">
          <Link as={routerLink} _hover={hoverStyles} to="/">
            О нас
          </Link>
          <Link as={routerLink} _hover={hoverStyles} to="/">
            Политика конфиденциальности
          </Link>
          <Link as={routerLink} _hover={hoverStyles} to="/">
            Связаться с нами
          </Link>
        </Flex>
      )}
      <Spacer />
      <WebsiteLogo isFooter={true} isLogged={isLogged && !isSignIn} />
    </Box>
  );
};

export default Footer;
