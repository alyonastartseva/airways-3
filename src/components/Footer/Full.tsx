import { RedButton } from "@/common/RedButton";
import { IDeparture, IReturn, TAdditionalServices } from "@/interfaces/footer";
import { Box, Heading, Text, Image, Flex } from "@chakra-ui/react";
import Dayjs from "dayjs";
import React, { FC } from "react";
import LogoSvg from "@images/LOGO.svg";

interface IProps {
	departure?: IDeparture;
	return?: IReturn;
	additional?: TAdditionalServices;
}

export const Full: FC<IProps> = ({ additional, departure, return: ret }) => {
	const { to, from, date, time, code, type, price, passenger } = departure!;

	const allAdditional = additional?.reduce((acc, item) => acc + item[1], 0);
	const total = additional
		? price + ret?.price! + allAdditional!
		: price + ret?.price!;

	const dateText = Dayjs(date).format("D MMM, ") + date.slice(0, 2);
	const dateText2 = ret && Dayjs(ret.date).format("D MMM, ") + date.slice(0, 2);
	return (
		<>
			<Flex>
				<Flex columnGap="2.875rem" fontWeight={"700"} fontSize={".875rem"}>
					<Flex columnGap="1.25rem">
						<Heading>Departure</Heading>
						<Heading>{`${from} - ${to}`}</Heading>
						<Box fontSize={".750rem"}>
							<Text
								position={"relative"}
								_before={{
									content: `""`,
									display: "inline-block",
									position: "absolute",
									left: "-1rem",
									top: ".56rem",
									w: "1rem",
									h: ".125rem",
									bg: "border",
									opacity: "0.6",
								}}
							>
								{dateText}
							</Text>
							<Text>{time}</Text>
							<Text>{type}</Text>
							<Text position={"relative"}>
								<Image position={"absolute"} left="-1.25rem" src={LogoSvg} />
								{code}
							</Text>
						</Box>
					</Flex>
					<Box h={"100%"} w="1px" bg={"#fff"}>
						{" "}
					</Box>
					{ret ? (
						<Flex columnGap="1.25rem">
							<Heading>Return</Heading>
							<Heading>{`${ret.from} - ${ret.to}`}</Heading>
							<Box fontSize={".750rem"}>
								<Text
									position={"relative"}
									_before={{
										content: `""`,
										display: "inline-block",
										position: "absolute",
										left: "-1rem",
										top: ".56rem",
										w: "1rem",
										h: ".125rem",
										bg: "border",
										opacity: "0.6",
									}}
								>
									{dateText2}
								</Text>
								<Text>{ret.time}</Text>
								<Text>{ret.type}</Text>
								<Text position={"relative"}>
									<Image position={"absolute"} left="-1.25rem" src={LogoSvg} />
									{ret.code}
								</Text>
							</Box>
						</Flex>
					) : (
						<Text color={"#EE5F5F"}>Select Return Flight</Text>
					)}
					{departure && ret && (
						<>
							<Box h={"100%"} w="1px" bg={"#fff"}>
								{" "}
							</Box>
							<Text marginLeft={"-1.5rem"} color={"#EE5F5F"}>
								See Price Details
							</Text>
						</>
					)}
				</Flex>
			</Flex>

			<Box>
				{additional && (
					<Flex w="80%">
						<Box w="60%">
							<Heading textAlign={"right"} fontWeight={"700"} as={"h3"}>
								Ticket Price
							</Heading>
							<Text textAlign={"right"} color="border">
								Flight Price{" "}
							</Text>
							<Text textAlign={"right"} color="border">
								Taxes and Fees{" "}
							</Text>
							<Heading textAlign={"right"} fontWeight={"700"} as={"h3"}>
								Additional Services
							</Heading>
							{additional!.map((e) => (
								<Text key={e[0]} textAlign={"right"}>
									{e[0]}
								</Text>
							))}
						</Box>
						<Box w="40%">
							<Heading textAlign={"right"} fontWeight={"700"} as={"h3"}>
								{price + ret?.price!}
							</Heading>
							<Text textAlign={"right"} color="border">
								{((price + ret?.price!) / 100) * 95}
							</Text>
							<Text textAlign={"right"} color="border">
								{((price + ret?.price!) / 100) * 5}
							</Text>
							<Heading textAlign={"right"} fontWeight={"700"} as={"h3"}>
								{allAdditional}
							</Heading>
							{additional!.map((e) => (
								<Text key={e[0]} textAlign={"right"}>
									{e[1]}
								</Text>
							))}
						</Box>
					</Flex>
				)}
				<Flex columnGap="1.5rem">
					{departure && ret && (
						<>
							<Box>
								<Text>Total price for {passenger} passengers</Text>
								<Text textAlign={"right"}>CHF {total}</Text>
							</Box>
							<RedButton
								text="Confirm"
								clickHandler={() => alert("Confirmed")}
							/>
						</>
					)}
				</Flex>
			</Box>
		</>
	);
};
