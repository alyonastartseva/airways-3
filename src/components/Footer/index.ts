export * from "@components/Footer/Footer";
export * from "@components/Footer/Full";
export * from "@components/Footer/Empty";
