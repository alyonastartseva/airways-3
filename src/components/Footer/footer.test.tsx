import { describe, expect, it } from "vitest";
import { render, screen, userEvent } from "@utils/test-utils";
import { departure, Footer, ret, additional } from "@components/Footer";

describe("Footer", () => {
	it("Footer without props", () => {
		const { container } = render(<Footer />);
		expect(container).toMatchSnapshot("0props");
	});

	it("Footer with 1 props", () => {
		const { container } = render(<Footer departure={departure} />);
		expect(container).toMatchSnapshot("1props");
	});

	it("Footer with 2 props", () => {
		const { container } = render(<Footer departure={departure} return={ret} />);
		expect(container).toMatchSnapshot("2props");
	});

	it("Footer with 3 props", () => {
		const { container } = render(
			<Footer departure={departure} return={ret} additional={additional} />,
		);
		expect(container).toMatchSnapshot("3props");
	});
});
