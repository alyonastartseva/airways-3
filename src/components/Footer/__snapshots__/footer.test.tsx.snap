// Vitest Snapshot v1

exports[`Footer > Footer with 1 props > 1props 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-k008qs"
    >
      <div
        class="css-13876xk"
      >
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Departure
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            ZRH - JFK
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              30 Dec, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 19:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <p
          class="chakra-text css-mx9oo2"
        >
          Select Return Flight
        </p>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="css-9s20yf"
      />
    </div>
  </div>
</div>
`;

exports[`Footer > Footer with 2 props > 2props 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-k008qs"
    >
      <div
        class="css-13876xk"
      >
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Departure
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            ZRH - JFK
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              30 Dec, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 19:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Return
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            JFK - ZRH
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              10 Oct, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 08:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <p
          class="chakra-text css-o2qitt"
        >
          See Price Details
        </p>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="css-9s20yf"
      >
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-0"
          >
            Total price for 
            1
             passengers
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            CHF 
            6200
          </p>
        </div>
        <button
          class="chakra-button css-bvuw0j"
          type="button"
        >
          Confirm
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Footer > Footer with 3 props > 3 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-k008qs"
    >
      <div
        class="css-13876xk"
      >
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Departure
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            ZRH - JFK
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              30 Dec, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 19:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Return
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            JFK - ZRH
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              10 Oct, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 08:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <p
          class="chakra-text css-o2qitt"
        >
          See Price Details
        </p>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="css-oxm8hx"
      >
        <div
          class="css-a6rva7"
        >
          <h3
            class="chakra-heading css-mf7hik"
          >
            Ticket Price
          </h3>
          <p
            class="chakra-text css-1d1r44r"
          >
            Flight Price
             
          </p>
          <p
            class="chakra-text css-1d1r44r"
          >
            Taxes and Fees
             
          </p>
          <h3
            class="chakra-heading css-mf7hik"
          >
            Additional Services
          </h3>
          <p
            class="chakra-text css-s2uf1z"
          >
            Seat Selection
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            Extra Baggage
          </p>
        </div>
        <div
          class="css-126n10b"
        >
          <h3
            class="chakra-heading css-mf7hik"
          >
            6200
          </h3>
          <p
            class="chakra-text css-1d1r44r"
          >
            5890
          </p>
          <p
            class="chakra-text css-1d1r44r"
          >
            310
          </p>
          <h3
            class="chakra-heading css-mf7hik"
          >
            2200
          </h3>
          <p
            class="chakra-text css-s2uf1z"
          >
            100
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            2100
          </p>
        </div>
      </div>
      <div
        class="css-9s20yf"
      >
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-0"
          >
            Total price for 
            1
             passengers
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            CHF 
            8400
          </p>
        </div>
        <button
          class="chakra-button css-bvuw0j"
          type="button"
        >
          Confirm
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Footer > Footer with 3 props > 3props 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-k008qs"
    >
      <div
        class="css-13876xk"
      >
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Departure
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            ZRH - JFK
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              30 Dec, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 19:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <div
          class="css-ky0lkx"
        >
          <h2
            class="chakra-heading css-0"
          >
            Return
          </h2>
          <h2
            class="chakra-heading css-0"
          >
            JFK - ZRH
          </h2>
          <div
            class="css-1jqojq2"
          >
            <p
              class="chakra-text css-1860ddu"
            >
              10 Oct, Fr
            </p>
            <p
              class="chakra-text css-0"
            >
              Departure 08:00
            </p>
            <p
              class="chakra-text css-0"
            >
              ECONOMY - Eco Light
            </p>
            <p
              class="chakra-text css-79elbk"
            >
              <img
                class="chakra-image css-1kjwwcr"
                src="/src/assets/images/LOGO.svg"
              />
              UA0018
            </p>
          </div>
        </div>
        <div
          class="css-x4tj1o"
        >
           
        </div>
        <p
          class="chakra-text css-o2qitt"
        >
          See Price Details
        </p>
      </div>
    </div>
    <div
      class="css-0"
    >
      <div
        class="css-oxm8hx"
      >
        <div
          class="css-a6rva7"
        >
          <h3
            class="chakra-heading css-mf7hik"
          >
            Ticket Price
          </h3>
          <p
            class="chakra-text css-1d1r44r"
          >
            Flight Price
             
          </p>
          <p
            class="chakra-text css-1d1r44r"
          >
            Taxes and Fees
             
          </p>
          <h3
            class="chakra-heading css-mf7hik"
          >
            Additional Services
          </h3>
          <p
            class="chakra-text css-s2uf1z"
          >
            Seat Selection
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            Extra Baggage
          </p>
        </div>
        <div
          class="css-126n10b"
        >
          <h3
            class="chakra-heading css-mf7hik"
          >
            6200
          </h3>
          <p
            class="chakra-text css-1d1r44r"
          >
            5890
          </p>
          <p
            class="chakra-text css-1d1r44r"
          >
            310
          </p>
          <h3
            class="chakra-heading css-mf7hik"
          >
            2200
          </h3>
          <p
            class="chakra-text css-s2uf1z"
          >
            100
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            2100
          </p>
        </div>
      </div>
      <div
        class="css-9s20yf"
      >
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-0"
          >
            Total price for 
            1
             passengers
          </p>
          <p
            class="chakra-text css-s2uf1z"
          >
            CHF 
            8400
          </p>
        </div>
        <button
          class="chakra-button css-bvuw0j"
          type="button"
        >
          Confirm
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Footer > Footer without props > 0 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-1x69zh0"
    >
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        About us
      </a>
      I
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        Terms and Conditions
      </a>
      I
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        Contact Us
      </a>
    </div>
    <div
      class="css-rk5plh"
    >
      <p
        class="chakra-text css-1vg6q84"
      >
        UX AIR APP
      </p>
      <div
        class="css-ky0lkx"
      >
        <a
          class="chakra-link css-0"
          href="#"
          target="_blank"
        >
          <img
            class="chakra-image css-0"
            src="/src/assets/images/appstore.png"
          />
        </a>
        <a
          class="chakra-link css-0"
          href="#"
          target="_blank"
        >
          <img
            class="chakra-image css-0"
            src="/src/assets/images/googleplay.png"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Footer > Footer without props > 0props 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-1x69zh0"
    >
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        About us
      </a>
      I
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        Terms and Conditions
      </a>
      I
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        Contact Us
      </a>
    </div>
    <div
      class="css-rk5plh"
    >
      <p
        class="chakra-text css-1vg6q84"
      >
        UX AIR APP
      </p>
      <div
        class="css-ky0lkx"
      >
        <a
          class="chakra-link css-0"
          href="#"
          target="_blank"
        >
          <img
            class="chakra-image css-0"
            src="/src/assets/images/appstore.png"
          />
        </a>
        <a
          class="chakra-link css-0"
          href="#"
          target="_blank"
        >
          <img
            class="chakra-image css-0"
            src="/src/assets/images/googleplay.png"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Footer > Footer without props 1`] = `
<div>
  <div
    class="css-3c5inh"
  >
    <div
      class="css-1x69zh0"
    >
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        About us
      </a>
      I
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        Terms and Conditions
      </a>
      I
      <a
        class="chakra-link css-0"
        href="#"
        target="_blank"
      >
        Contact Us
      </a>
    </div>
    <div
      class="css-rk5plh"
    >
      <p
        class="chakra-text css-1vg6q84"
      >
        UX AIR APP
      </p>
      <div
        class="css-ky0lkx"
      >
        <a
          class="chakra-link css-0"
          href="#"
          target="_blank"
        >
          <img
            class="chakra-image css-0"
            src="/src/assets/images/appstore.png"
          />
        </a>
        <a
          class="chakra-link css-0"
          href="#"
          target="_blank"
        >
          <img
            class="chakra-image css-0"
            src="/src/assets/images/googleplay.png"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
